Main/lights.ino

```diff
@@ -0,0 +1,92 @@
+
+void fadeToALevel(int fadeLevel) {
+	analogWrite(pin,fadeLevel);
+}
+
+void pellMorseCode() {
```

Comment:
This is so clear and easy to understand. I feel like everything is broken down really well so I can not only replicate this but understand it too! So cool!